src/styles/ForecastDetails.css

```diff
@@ -0,0 +1,10 @@
+.forecast-details {
+  margin-right: auto;
+  margin-left: auto;
+  width: 200px;
+  margin-top: 40px;
+}
+
+.forecast-details_boldText {
```

Comment:
Small point but the usual convention is `.class-name__element--modifier`, should have a double underscore here